go/libkb/wot.go

```diff
@@ -181,14 +192,23 @@ type apiWot struct {
 	Vouches []serverWotVouch `json:"webOfTrust"`
 }
 
-func fetchWot(mctx MetaContext, username *string) (res []serverWotVouch, err error) {
+type FetchWotVouchesArg struct {
+	Vouchee *string
+	Voucher *string
+}
```

Comment:
we aren't serializing this object, so if it's easier, i can't really think of a reason this couldn't be 
```
type FetchWotVouchesArg struct {
	Vouchee string
	Voucher string
}
```
we wouldn't have to fix those error logs, and then just instead of nil checks, we can do something like `len(arg.Voucher) > 0`.